internal/dbmigrate/003_river_job_tags_non_null.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE river_job ALTER COLUMN tags DROP NOT NULL,
```

Comment:
Doh, this is still kinda bad and could be changed, but you still need to add these to a list of migrations here along with `go:embed` statements:

https://github.com/riverqueue/river/blob/master/internal/dbmigrate/db_migrate.go#L47-L48

We should fix that, but it was the fastest/easiest way to introduce migrations at the time.